packages/js/src/settings/routes/site-representation.js

```diff
@@ -44,6 +45,11 @@ const SiteRepresentation = () => {
 	const mastodonPremiumLink = useSelectSettings( "selectLink", [], "https://yoa.st/get-mastodon-integration" );
 	const mastodonUrlLink = useSelectSettings( "selectLink", [], "https://yoa.st/site-representation-mastodon" );
 
+	const handleAddProfile = useCallback( async( arrayHelpers ) => {
```

Comment:
Why is this `async`?

Reply:
I didn't know this makes the calls afterwards be delayed, even if the function you await is not async. I'm used to the `setTimeout` with `0` to defer the execution. So new trick 😉